Earthfile

```diff
@@ -349,14 +349,19 @@ ipxe-iso:
                            mtools syslinux isolinux gcc-arm-none-eabi git make gcc liblzma-dev mkisofs xorriso
                            # jq docker
     WORKDIR /build
-    ARG ISO_NAME=${OS_ID}
+    ARG ISO_NAME=${OS_ID}        
+    COPY +version/VERSION ./
+    ARG VERSION=$(cat VERSION)
```

Comment:
I'm not sure of this - however versions where missing in released IPXE scripts

Reply:
I tried it and it works

Reply:
eeek... doesn't look like, opened #501